src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -16,19 +29,25 @@ export const TodoFilter = () => (
         type="text"
         className="input"
         placeholder="Search..."
+        value={filter.query}
+        onChange={e => setFilter({ ...filter, query: e.target.value })}
+
       />
```

Comment:
```suggestion
        onChange={e => setFilter({ ...filter, query: e.target.value })}
      />
```
It would be great to create separate functions to handle events